app/services/userService.js

```diff
@@ -54,7 +54,7 @@ class UserService {
       }
     });
 
-    return fetch(request).then(response => response.json());
+    return fetch(request);
```

Comment:
Good question.

As you may be guessing, this change has noting to do with webpack 2. I added this change post PR creation because while reviewing these changes with JC we found that I miss this in my previous PR. Then why I miss this?

In my previous PR I changed the server API for delete. Now it only returns a 200 status code with an empty body. I forget to update app code here where code is trying to parse as json an empty response. Removing the parse is the solution because we are only interested in a successful status code.